src/compute-client/src/controller/instance.rs

```diff
@@ -1044,9 +1039,7 @@ where
 
         // Validate sources have `since.less_equal(as_of)`.
         for source_id in dataflow.source_imports.keys() {
-            let storage_read_hold = self
-                .storage_controller
-                .acquire_read_hold_at_time(*source_id, as_of.clone())?;
```

Comment:
Is that method in `StorageController` now dead code? I didn't see the removal in this PR.

Reply:
Indeed, good catch!